### ISSUE TITLE
WCAG fail with Notify tabbing sequence

### ISSUE THREAD

**Issue by ethan-lp** [adaptlearning/adapt-contrib-core#140]
Notify pop ups are currently failing WCAG guidelines. Tabbing sequence should be contained to the modal pop up until closed. 
This containment should also include restricting access to browser elements i.e. Toolbar 

https://www.w3.org/WAI/ARIA/apg/patterns/dialogmodal/

**Comment by oliverfoster** [comment]
> This containment should also include restricting access to browser elements i.e. Toolbar 

I don't think this is the case. If the user wants to refresh the page or look up something else a content dialog should not prevent them from doing so, as with the mouse, one can move outside of a dialog, the expectation for the keyboard should be the same.

Secondarily, it's impossible to do this.

**Comment by ethan-lp** [comment]
Take a look at this @oliverfoster and let me know what you think

https://www.w3.org/WAI/ARIA/apg/example-index/dialog-modal/dialog.html

I understand there may be the need to refresh / change tab etc. but there are specific Keyboard shortcuts to do those things as well, regardless of a pop up present.

**Comment by oliverfoster** [comment]
I can see they've implemented it in the codepen but I can't see any reference to the spec.
We do actually have this behaviour in Adapt, but it's broken at the moment.

https://github.com/adaptlearning/adapt-contrib-core/blob/master/js/a11y/wrapFocus.js

This is missing a `tabindex="0"` https://github.com/adaptlearning/adapt-contrib-core/blob/a7546f63a027d0039a3f55d6d9e5affdd443d12c/js/helpers.js#L312

Personally, I think the tabbing should work exactly the same way for dialogs as it does for pages, in that the focus should circle around the browser UI when you leave the bottom of the content. I would consider that normal, standard, expected behaviour, trapping it is the exception.

**Comment by ethan-lp** [comment]
Hey @oliverfoster I tried changing this <a> within the DOM and ensuring _isPopupWrapFocusEnabled set to true and it is still cycling beyond the Popup. Am I missing anything else?

**Comment by oliverfoster** [comment]
![focuswrap](https://user-images.githubusercontent.com/7974663/174047385-90692b40-76fc-4fac-b927-1d808c840a7a.gif)

**Comment by ethan-lp** [comment]
Ah I see, apologies I had added the tabindex in the wrong place. 

Really appreciate that Oliver thank you!
